README.md

```diff
@@ -200,11 +200,12 @@ echo 8 > /sys/class/net/ib0/device/sriov_numvfs
 * `type` (string, required): "ib-sriov"
 * `deviceID` (string, required): A valid pci address of an InfiniBand SR-IOV NIC's VF. e.g. "0000:03:02.3"
 * `guid` (string, optional): InfiniBand Guid for VF.
-* `pkey` (string, optional): InfiniBand pkey for VF, this filed is used by [ib-kubernetes](https://www.github.com/Mellanox/ib-kubernetes) to add pkey with guid to InfiniBand subnet manager client e.g. [Mellanox UFM](https://www.mellanox.com/products/management-software/ufm), [OpenSM](https://docs.mellanox.com/display/MLNXOFEDv461000/OpenSM).
+* `pkey` (string, optional): InfiniBand pkey for VF, this field is used by [ib-kubernetes](https://www.github.com/Mellanox/ib-kubernetes) to add pkey with guid to InfiniBand subnet manager client e.g. [Mellanox UFM](https://www.mellanox.com/products/management-software/ufm), [OpenSM](https://docs.mellanox.com/display/MLNXOFEDv461000/OpenSM).
 * `ipam` (dictionary, optional): IPAM configuration to be used for this network, `dhcp` is not supported.
 * `link_state` (string, optional): Enforces link state for the VF. Allowed values: auto, enable, disable.
 * `rdmaIsolation` (boolean, optional): Enable RDMA network namespace isolation for RDMA workloads. More information
 about the system requirements to support this mode of operation can be found [here](https://github.com/Mellanox/rdma-cni)
+* `cniArgGUID` (bool, optional): Enforces ib-sriov-cni to get VF GUID from CNI-Args["mellanox.infiniband.app"], default to false. CNI-Args["mellanox.infiniband.app"] is configured by [ib-kubernetes](https://www.github.com/Mellanox/ib-kubernetes).
```

Comment:
working with ib-kubernetes we still allow to get GUID from CNI args. I think we need to change the name to IBKubernetesEnabled or something similar

Reply:
Done